supernode/daemon/mgr/ha/etcd_tool.go

```diff
@@ -0,0 +1,146 @@
+package ha
+
+import (
+	"context"
+	"fmt"
+	"github.com/dragonflyoss/Dragonfly/supernode/config"
+	"go.etcd.io/etcd/clientv3"
+	"go.etcd.io/etcd/mvcc/mvccpb"
+	"log"
+	"time"
+)
+
+type etcdMgr  struct{
+	config clientv3.Config
+	client *clientv3.Client
+	leaseTtl int64
+	leaseKeepAliveRsp <-chan *clientv3.LeaseKeepAliveResponse
+	host_ip  string//i  will try to use a already existing config struct to  replace it
+	leaseResp *clientv3.LeaseGrantResponse
+}
+
+//new a etcd tool object
+func NewEtcdMgr(cfg *config.Config)(*etcdMgr,error){
+	config:= clientv3.Config{
+		Endpoints:cfg.HaToolInfo,
+		DialTimeout: 10*time.Second,
+	}
+	// build connection to etcd
+	client,error:= clientv3.New(config)
```

Comment:
s/error/err
Since `error` is the key word of golang runtime. We do not  wish to make instance naming ambiguous.

Reply:
sorry, i have modified it